src/relayclient/ContractInteractor.ts

```diff
@@ -298,6 +298,15 @@ export default class ContractInteractor {
     return await this.web3.eth.getBlock(blockHashOrBlockNumber)
   }
 
+  validateAddress (address: string): string {
```

Comment:
I don't think that `validateAddress()` should return the address, it's confusing seeing `this.getCode(this.validateAddress(address)))`.